compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc

```diff
@@ -1,5 +1,7 @@
 == Deploy WAAS Out-Of-Band with VPC Traffic Mirroring
 
+NOTE: A minimum 22.06 version of BOTH console and Defenders is required for VPC Traffic Mirroring.
```

Comment:
VPC Traffic Mirroring requires a minimum 22.06 version of both the Console and Defenders.

Reply:
@divyabhushan - The sentence has been changed as per your suggestion